### PR TITLE
fix(CLI): pass required arguments to flashComplete message

### PR DESCRIPTION
**Pull request by jviotti**
The `flashComplete` message takes the drive object and the image
basename as arguments. This was updated on the GUI, but causes the CLI
to throw an error upon completion.

Change-Type: patch
Changelog-Entry: Fix "imageBasename is not defined" error on the CLI.
Signed-off-by: Juan Cruz Viotti <jv@jviotti.com>

### CODE REVIEW

File: lib/shared/messages.js [hunk 1]
```diff
@@ -34,7 +34,7 @@ module.exports = {
 
     flashComplete: _.template([
       '<%= imageBasename %> was successfully written to',
-      '<%= drive.description %> (<%= drive.name %>)'
+      '<%= drive.description %> (<%= drive.device %>)'
```
Comment (lurch) [inline lib/shared/messages.js]: Won't this change also affect the GUI too? I assume `drive.name` was previously used here for good reasons?
(I can never remember what the `.name` and `.device` fields map to for different OSes)

Comment (jviotti) [inline lib/shared/messages.js]: We use to manually add the `.name` property in the drive scanner, which contained the device path in Linux and macOS, and the list of mounted volumes joined by commas on Windows. In order to not duplicate that logic on the CLI, I moved the addition of the name property to `drivelist`, and upgraded it here.

File: lib/gui/modules/drive-scanner.js [hunk 2]
```diff
@@ -40,19 +40,6 @@ const availableDrives = Rx.Observable.timer(
     return Rx.Observable.fromNodeCallback(drivelist.list)();
   })
 
-  // Build human friendly "description"
-  .map((drives) => {
-    return _.map(drives, (drive) => {
-      drive.name = drive.device;
-
-      if (os.platform() === 'win32' && !_.isEmpty(drive.mountpoints)) {
```
Comment (jhermsmeier) [inline lib/gui/modules/drive-scanner.js]: Looks like the `require('os')` needs removing now:
```
etcher\lib\gui\modules\drive-scanner.js
  20:7  error  'os' is assigned a value but never used  no-unused-vars
```

Comment (jviotti) [inline lib/gui/modules/drive-scanner.js]: Oops, fixed!

**Comment by jviotti** [comment]
@jhermsmeier Is this good to merge?

**Comment by lurch** [comment]
Great! It's a shame that the commit-message didn't mention the bump of `drivelist` though ;)

File: lib/gui/modules/drive-scanner.js (hunk 2 above)
Comment (jhermsmeier) [inline lib/gui/modules/drive-scanner.js]: If I see this correctly, this block can now also be removed from https://github.com/resin-io/etcher/pull/1637, right @jviotti?

Comment (jviotti) [inline lib/gui/modules/drive-scanner.js]: Yep, we can get rid of it :)

**Pull request merged**